Detectors/TRD/calibration/src/CalibratorVdExB.cxx

```diff
@@ -13,22 +13,160 @@
 /// \author Ole Schmidt
 
 #include "TRDCalibration/CalibratorVdExB.h"
+#include "DataFormatsTRD/CalVdriftExB.h"
+#include "Fit/Fitter.h"
+#include "TStopwatch.h"
+#include "CCDB/CcdbApi.h"
+#include "CCDB/BasicCCDBManager.h"
+#include <string>
+#include <map>
 #include <memory>
 
+using namespace o2::trd::constants;
+
 namespace o2::trd
 {
 
+std::unique_ptr<TGraph> FitHelper::createTmpGraph(double vdFit, double laFit) const
+{
+  auto graph = std::make_unique<TGraph>();
+  double trdAnodePlane = .0335; // OS: what is this number? Is is actually needed?
```

Comment:
what about this one? is that in units of m?

Reply:
this is m; Alex says we should keep this value like that for the moment, we are still discussing the precise value.